bilge-impl/src/bitsize_internal.rs

```diff
@@ -71,7 +71,7 @@ fn generate_struct(struct_data: &ItemStruct, arb_int: &TokenStream) -> TokenStre
         }
         impl #ident {
             // #[inline]
-            #[allow(clippy::too_many_arguments, clippy::type_complexity)]
+            #[allow(clippy::too_many_arguments, clippy::type_complexity, clippy::unused_parens)]
```

Comment:
```suggestion
            #[allow(clippy::too_many_arguments, clippy::type_complexity, unused_parens)]
```
this is a `rustc` lint and is not suppressed with the `clippy` qualifier